src/post-submission-actions/program-enrollment-action.ts

```diff
@@ -25,7 +27,7 @@ export const ProgramEnrollmentSubmissionAction: PostSubmissionAction = {
       const payload: ProgramEnrollmentPayload = {
         patient: patient.id,
         program: programUuid,
-        dateEnrolled: enrollmentDate ? dayjs(enrollmentDate).format() : null,
+        dateEnrolled: isDate(enrollmentDate) ? dayjs(enrollmentDate).format() : encounterDate,
```

Comment:
Is it possible for `encounterDate` to be an object as opposed to a string?

Reply:
this line is supposed to handle instances where the enrollment date is not configured, which is the case in afe. The enrollment date is the encounter date but in normal post submissions from RFE, the field is configured